src/main/scala/mesosphere/marathon/api/CORSFilter.scala

```diff
@@ -0,0 +1,35 @@
+package mesosphere.marathon.api
+
+import javax.servlet._
+import javax.servlet.http.{ HttpServletRequest, HttpServletResponse }
+
+import scala.collection.JavaConverters._
+
+class CORSFilter extends Filter {
```

Comment:
Changing this to `class CORSFilter @Inject() (config: MarathonConfig)` will give you access to the config.